test/integration/narrow/routes/statistics.test.js

```diff
@@ -0,0 +1,70 @@
+const { countsPerStatus: mockCountsPerStatus } = require('../../../mocks/statistics')
+
+describe('Statistics endpoint', () => {
+  const createServer = require('../../../../app/server')
+  let server
+
+  jest.mock('../../../../app/repos/statistics')
+  const { getCountsPerStatus } = require('../../../../app/repos/statistics')
+
+  beforeEach(async () => {
+    jest.clearAllMocks()
+    server = await createServer()
+    await server.initialize()
+  })
+
+  describe('GET /statistics?queryName=countsPerStatus', () => {
+    test('route returns 200', async () => {
+      getCountsPerStatus.mockResolvedValue(mockCountsPerStatus)
+
+      const options = {
+        method: 'GET',
+        url: '/statistics?queryName=countsPerStatus'
+      }
+
+      const response = await server.inject(options)
+      expect(response.statusCode).toBe(200)
+    })
```

Comment:
could we add the response here...just be nice to have so it's easy to find if we are looking.  Think this would cover response schema question.